src/components/wallets/WalletsSearchInput.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <v-text-field
+    v-model.trim="computedSearch"
+    :label="t('wallets.search')"
+    :clearable="true"
+    :class="`${classes.root}__search`"
+    class="a-input"
+    color="primary"
+    :single-line="true"
+    type="text"
+    variant="underlined"
+    @input="$emit('change', computedSearch)"
+    @click:clear="$emit('change', computedSearch)"
+  >
+  </v-text-field>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const className = 'wallets-view'
+const classes = {
+  root: className
+}
+
+export default defineComponent({
+  setup() {
+    const { t } = useI18n()
+
+    // it is cannot be just a ref cause of vuetify bug: https://github.com/vuetifyjs/vuetify/issues/4144
+    const search = ref<string>('')
```

Comment:
```suggestion
    const search = ref('')
```

Type is inherited from default value